### ISSUE TITLE
About saving discriminator model

### ISSUE THREAD

**Issue by scey26** [thstkdgus35/EDSR-PyTorch#213]
Hello? Your amazing work is very nice reference for SR using deep learning.

**I wonder why your code do not saving the discriminator when using adversarial training mode.
(If i just do not find it, please let me know)

If i wanna retrain the model in adversarial training , i can not read the pretrained discriminator model, right?**

Then, why you don't consider it?

Thanks!
